docs/release-notes/snapcraft-8-7-0.rst

```diff
@@ -0,0 +1,146 @@
+.. _release-8.7.0:
+
+Snapcraft 8.7.0 release notes
+=============================
+
+03 March 2025
+
+Learn about the new features, changes, and fixes introduced in Snapcraft 8.7.0.
+
+
+Requirements and compatibility
+------------------------------
+
+See :ref:`System requirements<system-requirements>` for information on the minimum
+hardware and installed software.
+
+
+What's new
+----------
+
+Snapcraft 8.7.0 brings the following features, integrations, and improvements.
+
+
+Bash completion
+~~~~~~~~~~~~~~~
+
+We've updated the completion file for the Snapcraft snap, which provides command-line
+completion of commands and options in Bash-compatible shells. Try it out by
+typing ``snapcraft`` and pressing :kbd:`Tab` in your terminal.
+
+Previously, the completion file was outdated or incorrect because it had to be
+manually updated. Now it's generated dynamically, which means it will always be
+up-to-date with the current commands and options.
```

Comment:
Make the first sentence it's own paragraph and first.  The second sentence should be blended with the other paragraph.